backbone/src/main/java/org/researchstack/backbone/model/survey/ChoiceQuestionSurveyItem.java

```diff
@@ -8,7 +8,7 @@
 
 public class ChoiceQuestionSurveyItem extends QuestionSurveyItem<Choice> {
     /* Default constructor needed for serilization/deserialization of object */
-    ChoiceQuestionSurveyItem() {
+    public ChoiceQuestionSurveyItem() {
```

Comment:
Do we need to increase visibility of these constructors?

Reply:
Yes, otherwise there are no constructors for it (only GSON)